README.md

```diff
@@ -26,7 +26,13 @@ import { AuthProvider } from '@descope/react-sdk';
 
 const AppRoot = () => {
 	return (
-		<AuthProvider projectId="my-project-id">
+		<AuthProvider
+			projectId="my-project-id"
+			// If the Descope project manages the token response in cookies, a custom domain
+			// must be configured (e.g., https://auth.myapp.com)
+			// and should be set as the baseUrl property.
+			// baseUrl = "https://auth.myapp.com"
```

Comment:
"https://auth.app.myapp.com"

Reply:
see this thread https://authtown.slack.com/archives/C04490U8M7A/p1685539290752359?thread_ts=1685533466.671889&cid=C04490U8M7A

and https://docs.descope.com/manage/session/?search=base#manage-in-cookies-recommended-for-production

Reply:
auth.app.example.com